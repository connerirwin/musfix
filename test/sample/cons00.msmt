(declare-sort List 1)
(qualif eq ((x (List Int)))  (= x (List 2)))
