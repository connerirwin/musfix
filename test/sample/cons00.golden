
Preparing to find greatest fixpoint...

Inputs
--------
Reading from file test/sample/cons00.msmt
SortDecl "List" 1
Qualifier "eq" [Var (DataS "List" [IntS]) "x"] (Binary Eq (Var (DataS "List" [IntS]) "x") (Cons (DataS "List" [IntS]) "List" [IntLit 2]))

QMAP
--------
fromList []

Candidates
--------
[Candidate {solution = fromList [], validConstraints = fromList [], invalidConstraints = fromList [], label = "0"}]


Final candidates: 
0: [] (0 0)

